main.go

```diff
@@ -1,38 +1,61 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package main
 
 import (
 	"context"
-
-	"os"
-	"os/signal"
-
 	"github.com/Peripli/service-manager/env"
-	"github.com/Peripli/service-manager/sm"
-	"github.com/sirupsen/logrus"
+	"github.com/Peripli/service-manager/server"
+	"fmt"
+	"github.com/Peripli/service-manager/storage"
+	"github.com/Peripli/service-manager/storage/postgres"
+	"github.com/Peripli/service-manager/api"
+	cfenv "github.com/Peripli/service-manager/cmd/cf/env"
 )
 
 func main() {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-	handleInterrupts(ctx, cancel)
-
-	srv, err := sm.NewServer(ctx, env.Default())
+	ctx := context.Background()
+	server, err := CreateServer(ctx, cfenv.New(env.Default()))
 	if err != nil {
-		logrus.Fatal("Error creating the server", err)
+		panic(err)
 	}
-	srv.Run(ctx)
+	server.Run(ctx)
 }
 
-func handleInterrupts(ctx context.Context, cancelFunc context.CancelFunc) {
-	term := make(chan os.Signal)
-	signal.Notify(term, os.Interrupt)
-	go func() {
-		select {
-		case <-term:
-			logrus.Error("Received OS interrupt, exiting gracefully...")
-			cancelFunc()
-		case <-ctx.Done():
-			return
-		}
-	}()
-}
+// CreateServer creates service manager server
```

Comment:
this is not the right version of the main file to use - here no log cofiguration is done. Correct version of main should be: 
```
/*
 * Copyright 2018 The Service Manager Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package main

import (
	"context"
	"os"
	"os/signal"

	"github.com/Peripli/service-manager/env"
	"github.com/Peripli/service-manager/sm"
	"github.com/sirupsen/logrus"
)

func main() {
	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()
	handleInterrupts(ctx, cancel)

	srv, err := sm.NewServer(ctx, env.Default())
	if err != nil {
		logrus.Fatal("Error creating the server", err)
	}
	srv.Run(ctx)
}

func handleInterrupts(ctx context.Context, cancelFunc context.CancelFunc) {
	term := make(chan os.Signal)
	signal.Notify(term, os.Interrupt)
	go func() {
		select {
		case <-term:
			logrus.Error("Received OS interrupt, exiting gracefully...")
			cancelFunc()
		case <-ctx.Done():
			return
		}
	}()
}
```